report-model/src/main/proto/framework.proto

```diff
@@ -94,8 +94,6 @@ message MethodContext {
     map<string,string> parameters = 35;
     map<string,string> custom_contexts = 36;
     map<string,string> annotations = 37;
-    // A custom generated test name (e.a. cucumber scenario)
-    string test_name = 38;
```

Comment:
Why do you removed this? I think this is not a part of the context of this PR.

Reply:
Mistake